Source/WTF/wtf/PlatformEnable.h

```diff
@@ -216,6 +216,10 @@
 #define ENABLE_DESTINATION_COLOR_SPACE_LINEAR_SRGB 0
 #endif
 
+#if !defined(ENABLE_DIGITAL_CREDENTIALS)
+#define ENABLE_DIGITAL_CREDENTIALS 0
+#endif
```

Comment:
Is there a need for a build-time flag?

Reply:
Maybe not. I was unsure. I'll get rid of it.

Reply:
It seems like you only removed the PlatformEnable change.